index.html

```diff
@@ -32,23 +32,23 @@
         <option value="fashion">Fashion</option>
       </select>
     </div>
-    <div class="headlines">
+    
+    <div class="loading-gif">
+      <img src="./assets/images/ajax-loader.gif" class="loading">
     </div>
-  </div>
-  <!--Loading GIF-->
-  <div class="loading-gif">
-    <img src="./assets/images/ajax-loader.gif" class="loading">
-    <!--Footer-->
-    <footer>
-      <div class="copyright-footer">
-        <p class="copyright">© Copyright 2016 INSTANEWS</p>
-      </div>
-    </footer>
 
+    <div class="headlines"></div>
+  
   </div>
+    
+  <footer>
+    <div class="copyright-footer">
```

Comment:
this div seems really unnecessary esp. since you already have a footer element to work with.